assets/sass/app.scss

```diff
@@ -10,3 +10,15 @@
 .page-header {
   margin-bottom: $govuk-gutter;
 }
+
+.expanding-panel {
+  border-top: none;
+  &:last-child {
+    border-bottom: none;
+  }
+
+  .content {
+    padding: $govuk-gutter-half;
+    padding-left: 0;
+  }
+}
```

Comment:
this is targetting _every_ expanding panel - we almost certainly don't want that. If we actually _do_ want this then it should be added to the css in asl-components

Reply:
done